Teensy/src/gpio/gpio.cpp

```diff
@@ -60,7 +84,60 @@ static int32_t gpio_getGearPosition()
 
 static uint32_t gpio_getSteeringWheelPosition()
 {
-    return 0;
+    #include <Wire.h>
+    #include <Arduino.h>
+   const int MPU = 0x68; //MPU6050 I2C Address
```

Comment:
constants should be done outside of function. This is probably best put just above the function e.g. - 
```
const int STEERING_WHL_I2C_ADDR = 0x68;

static uint32_t gpio_getSteeringWheelPosition()
{
```

Also, try to be a little bit more descriptive in your variable name choices